lib/assets/main.css

```
@@ -0,0 +1,177 @@
+.app {
+  font-family: "Inter";
+
+  box-sizing: border-box;
+
+  --gray-50: #f8fafc;
+  --gray-100: #f0f5f9;
+  --gray-200: #e1e8f0;
+  --gray-300: #cad5e0;
+  --gray-400: #91a4b7;
+  --gray-500: #61758a;
+  --gray-600: #445668;
+  --gray-700: #304254;
+  --gray-800: #1c2a3a;
+
+  --blue-100: #ecf0ff;
+
+  --red--300: #F1A3A6;
+}
+
+input,
+select,
+textarea,
+button {
+  font-family: inherit;
+}
+
+.container {
+  border: solid 1px var(--gray-300);
+  border-radius: 0.5rem;
+  background-color: rgba(248, 250, 252, 0.3);
+  padding-bottom: 8px;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 8px 16px;
+  gap: 8px;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-start;
+  background-color: var(--blue-100);
+  padding: 8px 16px;
+  margin-bottom: 12px;
+  border-radius: 0.5rem 0.5rem 0 0;
+  border-bottom: solid 1px var(--gray-200);
+  gap: 16px;
+}
+
+.input {
+  padding: 8px 12px;
+  background-color: var(--gray-50);
+  font-size: 0.875rem;
+  border: 1px solid var(--gray-200);
+  border-radius: 0.5rem;
+  color: var(--gray-600);
+}
+
+input[required].empty {
+  border: 1px solid var(--red-300);
+}
+
+.input::placeholder {
+  color: var(--gray-400);
+}
+
+.input:focus {
+  outline: none;
+  border: 1px solid var(--gray-300);
+}
+
+.input--xs {
+  width: auto;
+  min-width: 150px;
+}
+
+.input--text {
+  max-width: 50%;
+}
+
+.input-label {
+  display: block;
+  margin-bottom: 2px;
+  font-size: 0.875rem;
+  color: var(--gray-800);
+  font-weight: 500;
+}
+
+.inline-input-label {
+  display: block;
+  margin-bottom: 2px;
+  color: var(--gray-600);
+  font-weight: 500;
+  padding-right: 6px;
+  font-size: 0.875rem;
+  text-transform: uppercase;
+}
+
+.field {
+  display: flex;
+  flex-direction: column;
+}
+
+.inline-field {
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+}
+
+.grow {
+  flex: 1;
+}
+
+.icon-button {
+  background: none;
+  border: none;
+  cursor: pointer;
+  padding: 4px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 100%;
+  color: var(--gray-500);
+  line-height: 1;
+}
+
+.icon-button:hover {
+  color: var(--gray-900);
+}
+
+.icon-button:focus {
+  outline: none;
+}
+
+.icon-button:disabled {
+  color: var(--gray-300);
+  cursor: default;
+}
+
+a.icon-button {
+  text-decoration: none;
+}
+
+.ri {
+  font-size: 1.25rem;
+  vertical-align: middle;
+  line-height: 1;
+}
+
+.help-box {
+  padding: 16px;
+  font-size: 0.875rem;
+  line-height: 1.5em;
+  color: var(--gray-700);
+  border-bottom: solid 1px var(--gray-200);
+}
+
+.help-box .section:not(:first-child) {
+  margin-top: 20px;
+}
+
+.help-box p {
+  margin: 0;
+  padding: 0;
+}
+
+.help-box h3 {
+  margin: 0;
+  margin-bottom: 8px;
+  font-size: 1.125em;
+}
+
+// For not showing hidden elements before AlpineJS loads
```

Unfortunately `//` are not valid CSS comments :)

```suggestion
/* For not showing hidden elements before AlpineJS loads */
```